cpp/ps0000/PS000.cpp

```diff
@@ -1,9 +1,10 @@
-#include <iostream>
+#include <iostream> // 2/2
 
 using namespace std;
 
-int main() // () not (void)
+int main() // 1/1
 {
-    cout << "Hello world!\n"; // don�t forget ;
-    return 0; // return 0 not 1
+    cout << "Hello, world!\n"; // don’t forget ; // 3/4
+    return 0; // return 0 not 1 // 1/1
 }
+// Great work 😎 9/10 ⭐

```

Comment:
- [ ] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit